.github/workflows/macos.yml

```diff
@@ -1,18 +1,27 @@
-name: macOS
-
 on:
   push:
     branches:
       - master
   pull_request:
-  schedule:
-    - cron: "0 6 * * *"
 
 jobs:
   build:
     runs-on: macos-latest
 
     steps:
       - uses: actions/checkout@v2
+
       - name: Install some packages
         run: ./test
+
+      - name: Test installed brew
+        run: brew ls ack
```

Comment:
What do you think about testing ALL installed packages by `./test`? What's missing right now:

1. `ag`
2. `git`
3. `bash`
4. `vlc`
5. `cask-fonts` tap

Reply:
sg